openreview/venue/invitation.py

```diff
@@ -1783,7 +1783,7 @@ def set_chat_invitation(self):
                         'writers': [venue_id, '${2/signatures}'],
                         'tag': {
                             'param': {
-                                'enum': ['👍', '👎', '😄', '😂', '🔥']
+                                'enum': ['👍', '👎', '👌', '👆', '😄', '😂', '🔥', '🚀', '✅']
```

Comment:
I'm not sure what the correct number of options to offer is. What do you think?

Reply:
I think that is ok for now.